pkg/minikube/bootstrapper/bsutil/kubeadm.go

```diff
@@ -68,6 +68,12 @@ func GenerateKubeadmYAML(cc config.ClusterConfig, n config.Node, r cruntime.Mana
 		}
 		return nil, errors.Wrap(err, "getting cgroup driver")
 	}
+	// TODO: investigate why containerd (v1.6.15) does not work with k8s (v1.25.3) when both are set to use systemd cgroup driver
```

Comment:
Could you create an issue for this if not already done and link the issue here

Reply:
sure - i've created https://github.com/kubernetes/minikube/issues/15633 with more details and logs there, and referenced it in the comment